tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Java.cs

```diff
@@ -77,7 +77,7 @@ private static void UpdatePackageVersions(string projectFile, IDictionary<string
         public override async Task<IterationResult> RunAsync(string project, string languageVersion,
             IDictionary<string, string> packageVersions, string testName, string arguments, string context)
         {
-            var processArguments = $"-XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75 -jar {context} -- {testName} {arguments}";
+            var processArguments = $"-XX:InitialRAMPercentage=75.0 -XX:MaxRAMPercentage=75.0 -jar {context} -- {testName} {arguments}";
```

Comment:
Maybe to play safe initial could be something less like 50%?

Reply:
The agent doesn't do much else during runs.